data/final/payment-methods.json

```diff
@@ -131,6 +131,32 @@
     ],
     "capabilities": []
   },
+  {
+    "id": "bancontact-mobile",
+    "type": "online",
+    "name": "Bancontact app",
+    "images": {
+      "small": {
+        "url": "https://flowcdn.io/util/logos/payment-methods/bancontact/30/original.png",
+        "width": 65,
+        "height": 41
```

Comment:
These heights are incorrect, however this seems to be standard across all payment method images. @RoryH Do you have any context here?